src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -2,6 +2,28 @@ import Cookies from 'universal-cookie';
 import jwtDecode from 'jwt-decode';
 
 
+function getAuthenticatedUserFromDecodedAccessToken(decodedAccessToken) {
+  if (decodedAccessToken === null) {
+    throw new Error('Decoded access token is required to get authenticated user.');
+  }
+
+  const authenticatedUser = {
+    userId: decodedAccessToken.user_id,
+    username: decodedAccessToken.preferred_username,
+    roles: decodedAccessToken.roles ? decodedAccessToken.roles : [],
+    administrator: decodedAccessToken.administrator,
+  };
+  return authenticatedUser;
+}
+
+function ensureAuthenticatedResponse(decodedAccessToken) {
```

Comment:
`formatAuthenticatedResponse` or something like that?  Using the word 'ensure' makes me think it's sorta a peer method of ensureAuthenticatedUser/will redirect on a non-authenticated user, or something.